### PR TITLE
feat: jetson nano SoC

### PR DESCRIPTION
**Pull request by frezbo**
Support Jetson Nano SoC

Depends on https://github.com/talos-systems/pkgs/pull/380

Signed-off-by: Noel Georgi <git@frezbo.dev>

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/4819)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -403,6 +397,8 @@ COPY --from=pkg-libressl-arm64 / /rootfs
 COPY --from=pkg-libseccomp-arm64 / /rootfs
 COPY --from=pkg-linux-firmware-arm64 /lib/firmware/bnx2 /rootfs/lib/firmware/bnx2
 COPY --from=pkg-linux-firmware-arm64 /lib/firmware/bnx2x /rootfs/lib/firmware/bnx2x
+COPY --from=pkg-linux-firmware-arm64 /lib/firmware/rtl_nic /rootfs/lib/firmware/rtl_nic
```
Comment (smira) [inline Dockerfile:400]: do we need this in the rootfs?

File: website/content/docs/v0.15/Single Board Computers/jetson_nano.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+Download the latest `talosctl`.
+
+```bash
+curl -Lo /usr/local/bin/talosctl https://github.com/talos-systems/talos/releases/latest/download/talosctl-$(uname -s | tr "[:upper:]" "[:lower:]")-amd64
+chmod +x /usr/local/bin/talosctl
+```
+
+## Flashing the firmware to on-board SPI flash
+
+We will use the [R32.6.1 release](https://developer.nvidia.com/embedded/l4t/r32_release_v6.1/t210/jetson-210_linux_r32.6.1_aarch64.tbz2) for the Jetson Nano.
```
Comment (smira) [inline website/content/docs/v0.15/Single Board Computers/jetson_nano.md:23]: I don't know if it's true, but I guess flashing firmware is one-time procedure? we don't need to flash it every time?

(does it makes sense to state it in the docs?)

File: Dockerfile (hunk 1 above)
Comment (frezbo) [reply Dockerfile:400]: yes, this driver is loaded from the `rootfs` and if I remove I see warning about failing to load

File: website/content/docs/v0.15/Single Board Computers/jetson_nano.md (hunk 2 above)
Comment (frezbo) [reply website/content/docs/v0.15/Single Board Computers/jetson_nano.md:25]: good catch, fixed

**Review by smira — APPROVED** [review]

**Pull request merged**